ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -3,6 +3,7 @@
 REPOS_DIR=/etc/yum.repos.d
 DISTRO_NAME=centos7
 LSB_RELEASE=redhat-lsb-core
+read -r -a inst_rpms <<< "$INST_RPMS"
```

Comment:
(lint) Possible misspelling: INST_RPMS may not be assigned, but inst_rpms is. [SC2153]